Feat/cirrus 753/parametise outcome spotlight

kik-kik opened the pull request:
Feat/cirrus 753/parametise outcome spotlight

This is version 1 of the change. This does not currently support multiple message_ids and branches. This will come as a separate change.

Please let me know your thoughts @scholtzan

scholtzan commented on jetstream/config.py:
```diff
@@ -478,18 +489,23 @@ def from_dict(cls, d: dict) -> "MetricsSpec":
+        self,
+        spec: "AnalysisSpec",
+        experiment: ExperimentConfiguration,
+        config_parameters: Optional[Dict[Any, Any]] = dict(),
```
I think we should be consistent with naming
```suggestion
       parameters: Optional[Dict[Any, Any]] = dict(),
```

scholtzan commented on jetstream/tests/conftest.py:
```diff
@@ -246,6 +261,31 @@ def fake_outcome_resolver(monkeypatch):
         """
     )
 
+    parametised_config = dedent(
```
typo
```suggestion
    parameterized_config = dedent(
```

scholtzan reviewed (approved):
The config parsing logic looks good 👍

kik-kik commented:
@scholtzan made the requested changes. Just for future reference, is it fair for me to assume if you approve I'm okay to merge after addressing your comments?

scholtzan commented:
yes
